File: WebAppVen/Scripts/app/model/MmWhapldt.js
Ext.define('WEBAPP.model.MmWhapldt', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'APPLY_DATE', type: 'string' },
        { name: 'APPLY_YEAR_MONTH', type: 'string' },
        { name: 'APPLY_DAY', type: 'string' },
        { name: 'WH_NO', type: 'string' },
        { name: 'WH_NO_N', type: 'string' },
        { name: 'WH_NO_OLD', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});

File: WebApp/Models/MM/MM_WHAPLDT.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class MM_WHAPLDT : JCLib.Mvc.BaseModel
    {
        public string APPLY_DATE { get; set; }
        public string APPLY_YEAR_MONTH { get; set; }
        public string APPLY_DAY { get; set; }
        public string WH_NO { get; set; }
        public string WH_NO_N { get; set; }
        public string WH_NO_OLD { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }

        public string APPLY_TYPE { get; set; }
    }
}